testdata/p4_16_samples_outputs/parser-unroll/parser-unroll-test1-midend.p4

```diff
@@ -43,6 +43,7 @@ parser MyParser(packet_in packet, out headers hdr, inout metadata meta, inout st
     @name("MyParser.index") int<32> index_0;
     state stateOutOfBound {
         verify(false, error.StackOutOfBounds);
+        transition reject;
```

Comment:
this is unreachable, verify(false) will always transition to reject.
https://p4.org/p4-spec/docs/P4-16-v1.2.2.html#sec-verify
So this change is really unnecessary.